### ISSUE TITLE
Error on start up - 1.13

### ISSUE THREAD

**Issue by Sheepotastic** [TheBusyBiscuit/Slimefun4#835]
## Description (Required)
When running the plugin, 90% of things don't load, including items and entire categories. I also get an error on startup, but being as it was on start up, I only just noticed it. I've provided pictures in the evidence section. I get no further errors, just the one on startup. This has been a persistent problem through multiple updates.

## Server Log
https://pastebin.com/pvCx88rJ

## Evidence / Proof (Required)
https://i.imgur.com/DavCgf4.png
As shown, various sections are missing, of these sections, many of the items don't show up or work properly.

## Environment (Required)
Minecraft - 1.13
SF Version - vDev - 65
Craftbukkit - v1_1_R1
CS-CoreLib - vDev - 33

**Comment by dart737** [comment]
damn, over 2 weeks and no response or assigning the problem

**Comment by WalshyDev** [comment]
This issue still persist on newer builds? If not, please close this issue.

**Comment by Sheepotastic** [comment]
I honestly don't know. This problem took so long to fix that I just downgraded my server to 1.12.2 anyways. This and others of my more important plugins refused to have working updates, so I ultimately decided we'd ultimately gain more content than the 1.13 update offered, via the plugins that worked on 1.12.
This was still an issue as of about a months ago and I got no response of if it was ever really addressed.
I'll close it and assume it was fixed more recently.
